### ISSUE TITLE
iOS build: Reached end of file while looking for: uint32_t

### ISSUE THREAD

**Issue by sanja-sa** [godotengine/godot#7687]
**Operating system or device - Godot version:**
Mac 10.11.6,  Xcode 8.2.1, Godot 2.1.2 (official), Godot 2.1.1(official)

**Issue description:**
Try to build for iOS (success) and run on device:
"XCode error : Reached end of file while looking for: uint32_t"

To correct this bug (or features):
Rename "Godot_v2.1.2-stable_export_templates.tpz" -> "Godot_v2.1.2-stable_export_templates.zip"
Unpack "GodotiOSXCode.zip"
Go to upacked folder "godot_ios_xcode"
Copy file "godot_ios_xcode/godot_opt.iphone" to folder "godot_ios_xcode/godot_xcode" and replace "godot_opt.iphone" (with 0 size)
And copy your "data.pck" file in the same folder to replace "data.pck" (with 0 size)

Then try build and then run build.
All done

**Steps to reproduce:**

**Link to minimal example project:**
:

**Comment by bojidar-bg** [comment]
Uhm, why was this closed?

**Comment by punto-** [comment]
This is an issue with the template, the xcode project points to the wrong
executable somehow

**Comment by florix91** [comment]
Why this isn't closed? The problem has been fixed if i remember correctly

**Comment by akien-mga** [comment]
This isn't closed because nobody said it had been fixed. Can you confirm it works with Xcode 8 and Godot 2.1.2-stable?

**Comment by volzhs** [comment]
I tested it recently, and confirmed ios template is wrong.
`godot_debug.iphone` and `godot_opt.iphone` are in 2 places.

```
/godot_ios_xcode/godot_debug.iphone
/godot_ios_xcode/godot_opt.iphone
/godot_ios_xcode/godot_xcode/godot_debug.iphone (0 bytes)
/godot_ios_xcode/godot_xcode/godot_opt.iphone (0 bytes)
```

I need to move `godot_debug[opt].iphone` from `/godot_ios_xcode/` to `/godot_ios_xcode/godot_xcode/`.

**Comment by florix91** [comment]
I tough Ariel had fixed it, maybe she forgot. However do we need the correct files in the two different folders or only where those 0 bytes files are?

**Comment by akien-mga** [comment]
Fixed in 2.1.3.
